### PR TITLE
DigiDNA Xcode 14

### PR DESCRIPTION
**Pull request by relgit**
This PR implements the changes introduced with Xcode 14.

### CODE REVIEW

File: Manifests/ManifestsApple/com.apple.dnsProxy.managed.plist [hunk 1]
```diff
@@ -182,6 +182,22 @@ The payload organization for a payload need not match the payload organization i
...
+		    <string>DNSProxyUUID</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>iOS</string>
+			</array>
+		    <key>pfm_title</key>
+		    <string>DNS Proxy UUID</string>
+		    <key>pfm_type</key>
+		    <string>string</string>
+		</dict>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.dnsProxy.managed.plist:200]: spacing got a little funky

File: Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2022-02-09T09:11:11Z</date>
+	<date>2022-09-05T01:03:41Z</date>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist:20]: bump pfm_version as well?

File: Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist [hunk 3]
```diff
@@ -15,7 +15,7 @@
 		<key>pfm_interaction</key>
 		<string>combined</string>
 		<key>pfm_last_modified</key>
-		<date>2022-05-25T08:04:43Z</date>
+		<date>2022-09-05T01:03:41Z</date>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist:18]: bump pfm_version as well?

File: Manifests/ManifestsApple/com.apple.systempreferences.plist [hunk 4]
```diff
@@ -642,6 +645,127 @@ The payload organization for a payload need not match the payload organization i
...
+						<string>HeadphoneSettings</string>
+						<string>Internet-Accounts-Settings</string>
+						<string>Keyboard-Settings</string>
+						<string>Localization-Settings</string>
+						<string>Lock-Screen-Settings</string>
+						<string>LoginItems-Settings</string>
+						<string>Mouse-Settings</string>
+						<string>Network-Settings</string>
+						<string>NetworkExtensionSettingsUI</string>
+						<string>Notifications-Settings</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.systempreferences.plist:739]: Can we replace the hyphens with spaces pls?

**Review by apizz — CHANGES_REQUESTED** [review]
Just a few minor fixes.  Thanks for this work, as always :)

File: Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist [hunk 5]
```diff
@@ -230,6 +240,20 @@ A profile can consist of payloads with different version numbers. For example, c
...
+				<key>pfm_default</key>
+				<true/>
+				<key>pfm_name</key>
+				<string>allowAirPlayIncomingRequests</string>
+				<key>pfm_title</key>
+				<string>Allow incoming AirPlay requests</string>
+				<key>pfm_supervised</key>
+				<true/>
+				<key>pfm_macos_min</key>
+				<string>12.3</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.applicationaccess-macOS.plist:253]: my OCD 😅 can we alphabetize the key names listed in the dict?

File: Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist (hunk 2 above)
Comment (relgit) [reply Manifests/ManifestsApple/com.apple.applicationaccess-tvOS.plist:20]: Keys weren't added or removed in this manifest.

**Review by apizz — APPROVED** [review]
lgtm!

**Pull request merged**